.vscode/settings.json

```diff
@@ -47,5 +46,6 @@
         "xml",
         "msbuild",
         "javascript"
-    ]
+    ],
+    "dotnet.formatting.organizeImportsOnFormat": true
```

Comment:
Just calling attention to this, I know I brought it up before but I'll leave it to you if you want to take this change.